cmd/git-test/main.go

```diff
@@ -0,0 +1,75 @@
+package main
```

Comment:
Do we need this test file in the master? I would suggest to remove it now that the problem is fixed.